### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by TommyM0ntana**

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# pull request under review by Forison

**Review by Forison — CHANGES_REQUESTED** [review]
## Good Job @TommyM0ntana 👋
I nearly felt I was on the original [newsweek](newsweek.com) website.🙂 You have done a great job, so far.
let us do these few changes so we can have our project approved.🙂

* [ ] The titles(top stories, features stories) are not seen at MD, SM, XS.🙂
This is how it looks currently ⬇️

Screenshot from 2019-10-17 18-02-43
Screenshot from 2019-10-17 18-02-54
 This is a result of the fixed nature of the navbar. make sure you have given appropriate margin-top(50-70px) to``` <main>```  to avoid this.

* [ ] let us make the subscribe button better by giving it this class ```.btn btn-light```🙂

* [ ] currently project ends at "FEATURED SLIDESHOWS". Let us add the remaining elements as found on the [newsweek](newsweek.com) page🙂

* [ ] let us fix the images in the magazine section. This is how they look like currently at  ```xs``` and ```sm```⬇️

Screenshot from 2019-10-17 18-13-10
Screenshot from 2019-10-17 18-13-17

this is because you gave your image height. Set the height to auto and give it bootstrap class ```img-responsive``` 

* [ ] Let us add these parts to our project ⬇️

Screenshot from 2019-10-17 18-23-12
Screenshot from 2019-10-17 18-23-19
Screenshot from 2019-10-17 18-23-23

refer to [newsweek](newseek.com) check how these elements behave at each breakpoint

* [ ] let us check our footer and make changes to it  🙂
Screenshot from 2019-10-17 18-28-13
refer to [newsweek](newseek.com) check how this element behave at each breakpoint

* note: > let us try to have the elements in each section behave as it does on the [Newsweek (newsweek.com) page

## Kindly do these changes and submit the project for review

feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. i would be happy to assist you.

Happy coding😇😇😇

File: style.css [hunk 1]
```diff
@@ -0,0 +1,602 @@
...
+
+img {
+  width: 100%;
+}
+
+/* NAVBAR */
+.top-footer-a-container {
+  display: inline-block;
+  flex: 1;
+  text-align: right;
```
Comment (Forison) [inline style.css:25]: you can use ```.text-right``` bootsrap class

**Comment by shloch** [comment]
Hello @TommyM0ntana 
You can approve this Pull Request

Corrections have been made, GREAT JOB 👏👏👏👏

Happy Coding for your next step

**Pull request merged**